github/ci/testgrid/gen-config.yaml

```diff
@@ -2,6 +2,7 @@ dashboards:
 - name: kubevirt-containerized-data-importer-presubmits
 - name: kubevirt-presubmits
 - name: kubevirt-periodics
+- name: kubevirt-postsubmits
```

Comment:
Is it worth adding a kubevirt-postsubmits panel to testgrid just for email notifications here? We have alerts in slack already - maybe we could trigger slack to send an email for for failed postsubmits?

Reply:
I'd rather not have added the kubevirt-postsubmits dashboard, but unfortunately the testgrid config does not allow that.

And yes, I know that we have slack notifications, however the complaint was that people missed that and they wanted the email.

I think failure of a release job is important enough to have another dashboard, if you then get the email alert as a second channel. WDYT?

Reply:
Ok thats fair enough for me.